student/src/main/webapp/Shared/CSS/Universal/shell.css

```diff
@@ -27,8 +27,8 @@ body {
 	overflow:hidden;
```

Comment:
Below commit is the actual code change
https://github.com/SmarterApp/TDS_Student/commit/7578c162c4a582e1d1d71b92e6763d57fcbfd9d0?diff=split